test/bdd/fixtures/hub-router/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       - ${HUB_ROUTER_DIDCOMM_HTTP_PORT}:${HUB_ROUTER_DIDCOMM_HTTP_PORT}
       - ${HUB_ROUTER_DIDCOMM_WS_PORT}:${HUB_ROUTER_DIDCOMM_WS_PORT}
     entrypoint: ""
-    command: /bin/sh -c "hub-router start"
+    command: /bin/sh -c "cp /etc/tls/* /usr/local/share/ca-certificates/;update-ca-certificates; hub-router start"
```

Comment:
We typically mount the CA certs into a volume and pass them as configuration options to the application. I cannot say with certainty why we're doing it that way but I like this method since it does not require extra code on our part.

Reply:
The host OS' configuration is never touched because we always run the binaries inside docker containers.

Reply:
@llorllale when we communicate between two services(in docker) there is a problem with invalid certs. This change makes the certs `trustable`.
I've got this error in `edge-agent`. Routers are not able to communicate with each other due to ` invalid certs` error.

Reply:
@soluchok that sounds about right. Sorry if my comment distracted from the motivation behind this PR. Maybe you should include that comment in this PR's description.

My comment above was just highlighting how we configure the CA certs differently, but I like this way better because it is effective and involves less work.